app/about_us/page.module.css

```diff
@@ -1,5 +1,40 @@
 .app {
-  background-color: black;
-  width: 100vw;
-  height: 100vh;
+    background-color: black;
+    width: 100vw;
+    height: 100vh;
+}
+
+/* Shared Styles for Background Image */
+.fullBGpicture::before {
+    content: "";
+    position: fixed;
+    top: 0;
+    bottom: 0;
+    left: 0;
+    right: 0;
+    width: 100%;
+    height: 100%;
+    z-index: -1;
+    background-color: #000;
+    background-size: cover;
+    background-repeat: no-repeat;
+}
+
+/* Background Images for Different Screen Sizes */
+@media screen and (min-width: 1200px) {
+    .fullBGpicture::before {
+        background-image: url("/crew/background-crew-desktop.jpg");
+    }
+}
+
+@media screen and (min-width: 768px) and (max-width: 1199px) {
+    .fullBGpicture::before {
+        background-image: url("/crew/background-crew-tablet.jpg");
```

Comment:
Hahaha, I was too lazy to do this. Amazing!

Reply:
This could be moved to the `globals.css` file, because I used the same background in different places